src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.jsx

```diff
@@ -77,20 +95,34 @@ export default class InstalledIntegration extends React.Component {
           <Box px={2} flex={1} style={style}>
             <IntegrationItem integration={integration} />
           </Box>
-          {this.hasConfiguration() && (
+          {integration.status === 'active' &&
+          this.hasConfiguration() && (
             <Box mr={1}>
               <Button
                 size="small"
-                to={`/settings/${orgId}/integrations/${provider.key}/${integration.id}/`}
-              >
+                to={`/settings/${orgId}/integrations/${provider.key}/${integration.id}/`}>
                 {t('Configure')}
               </Button>
             </Box>
           )}
+          {integration.status === 'disabled' && (
+            <Box mr={1}>
+              <AddIntegrationButton
+                size="small"
+                priority="danger"
+                provider={provider}
+                integration={integration}
+                onAddIntegration={this.mergeIntegration}
```

Comment:
Yeah this should call something like `this.props.reinstallIntegration` and then that will be set up a component to `mergeIntegration`